api/src/main/java/org/xnio/_private/Messages.java

```diff
@@ -194,6 +194,9 @@
     @Message(id = 40, value = "Mismatched IP address type; expected %s but got %s")
     IllegalArgumentException mismatchAddressType(Class<? extends InetAddress> expected, Class<? extends InetAddress> actual);
 
+    @Message(id = 41, value = "'%s' is not a valid Strength value")
+    IllegalArgumentException invalidStrength(String name);
```

Comment:
It seems like this is not used by anything?

Reply:
it's used here https://github.com/xnio/xnio/pull/132/files#diff-3e9c603fc240e98742b000bbe00edd55R64